velox/docs/functions/spark/conversion.rst

```diff
@@ -99,6 +99,25 @@ Valid examples
   SELECT cast(cast(2147483648.90 as DECIMAL(12, 2)) as integer); -- -2147483648
   SELECT cast(cast(2147483648.90 as DECIMAL(12, 2)) as bigint); -- 2147483648
 
+From timestamp
+^^^^^^^^^^^^^
+
+Casting a timestamp to an integer returns the number of seconds since the epoch (1970-01-01 00:00:00 UTC).
```

Comment:
Perhaps also mention how the value is rounded.

Reply:
done